N/ngspice/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "ngspice"
-version = v"37"
+version = v"39.3"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/imr/ngspice.git", "c4efe2e3ac264b6889e844f935410f9a795f1a68"), #ngspice git mirror
-    DirectorySource("./bundled")
+    GitSource("https://github.com/imr/ngspice.git", "f73d3b20a07cd206db8d7c7dd8679e6eecb02391"), #ngspice git mirror
+    DirectorySource(joinpath(@__DIR__,"./bundled"))
```

Comment:
This wasn't a great idea: https://github.com/JuliaBinaryWrappers/ngspice_jll.jl/commit/4b98f0685b18567b3a8311cfe501acaeea387a9f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22

Reply:
Yeah i wasn't sure if this would work...